src/Spectre.System/IO/Directory.cs

```diff
@@ -64,8 +64,22 @@ public IEnumerable<IDirectory> GetDirectories(string filter, SearchScope scope)
         public IEnumerable<IFile> GetFiles(string filter, SearchScope scope)
         {
             var option = scope == SearchScope.Current ? SearchOption.TopDirectoryOnly : SearchOption.AllDirectories;
-            return _directory.GetFiles(filter, option)
-                .Select(file => new File(new FilePath(file.FullName)));
+            IEnumerable<IFile> files = Array.Empty<IFile>();
+            files = files.Concat(_directory.GetFiles(filter, SearchOption.TopDirectoryOnly)
```

Comment:
We could change this to: 

```
var files = _directory.GetFiles(filter, SearchOption.TopDirectoryOnly)
    .Select(file => new File(new FilePath(file.FullName)))
```

Reply:
Mostly. I merged your suggestion:
```
            IEnumerable<IFile> files = _directory.GetFiles(filter, SearchOption.TopDirectoryOnly)
                .Select(file => new File(new FilePath(file.FullName)));
```

I have to keep `IEnumerable<IFile>` to honor the method return value.